docs/user/security/authentication/index.asciidoc

```diff
@@ -383,21 +383,11 @@ xpack.security.authc.providers:
 
 One of the most popular use cases for anonymous access is when you embed {kib} into other applications and don't want to force your users to log in to view it. If you configured {kib} to use anonymous access as the sole authentication mechanism, you don't need to do anything special while embedding {kib}.
 
-If you have multiple authentication providers enabled, and you want to automatically log in anonymous users when embedding, then you will need to add the `auth_provider_hint=<anonymous-provider-name>` query string parameter to the {kib} URL that you're embedding.
+If you have multiple authentication providers enabled, and you want to automatically log in anonymous users when embedding dashboards or visualizations, then toggle `Public URL` switch presented in the embedding menu.
```

Comment:
> If you have multiple authentication providers enabled, and you want to automatically log in anonymous users when embedding dashboards, visualizations, and saved searches, then toggle *Public URL* in the *Share > Embed code* menu in  *Dashboard*, *Visualize*, or *Discover*.

Thanks, but will omit saved searches and `Discover` since we don't support embedding for them.

Reply:
++